### ISSUE TITLE
OnKnockout event issue

### ISSUE THREAD

**Issue by llde** [llde/xOBSE#68]
I've found a rather strange issue with both the OnKnockout block and event handler. It's said that OnKnockout triggers once when the actor is knocked out. However, if the actor is hit by magic damage (spell or enchanted weapon) it's triggered non-stop until the the actor gets back up. I first though it was the event handler working incorrectly, but after further testing discovered the issue is also true for the OnKnockout block. Unless I'm missing something, this is a bug in the game itself, but it would be nice to have the event handler trigger only once when the actor is knocked down.
